pkg/network/ebpf/conntrack.go

```diff
@@ -0,0 +1,66 @@
+//+build linux
+
+package ebpf
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+
+	"github.com/DataDog/datadog-agent/pkg/process/util"
+)
+
+// Family returns whether a tuple is IPv4 or IPv6
+func (t ConntrackTuple) Family() ConnFamily {
+	if t.Metadata&uint32(IPv6) != 0 {
+		return IPv6
+	}
+	return IPv4
+}
+
+// Type returns whether a tuple is TCP or UDP
+func (t ConntrackTuple) Type() ConnType {
+	if t.Metadata&uint32(TCP) != 0 {
+		return TCP
+	}
+	return UDP
+}
+
+// SourceAddress returns the source address
+func (t ConntrackTuple) SourceAddress() util.Address {
+	if t.Metadata&uint32(IPv6) != 0 {
+		return util.V6Address(t.Saddr_l, t.Saddr_h)
+
+	}
+	return util.V4Address(uint32(t.Saddr_l))
+}
+
+// SourceEndpoint returns the source address and source port joined
+func (t ConntrackTuple) SourceEndpoint() string {
+	return net.JoinHostPort(t.SourceAddress().String(), strconv.Itoa(int(t.Sport)))
+}
+
+// DestAddress returns the destination address
+func (t ConntrackTuple) DestAddress() util.Address {
+	if t.Metadata&uint32(IPv6) != 0 {
+		return util.V6Address(t.Daddr_l, t.Daddr_h)
+
+	}
+	return util.V4Address(uint32(t.Daddr_l))
+}
+
+// DestEndpoint returns the destination address and source port joined
+func (t ConntrackTuple) DestEndpoint() string {
+	return net.JoinHostPort(t.DestAddress().String(), strconv.Itoa(int(t.Dport)))
+}
+
+func (t ConntrackTuple) String() string {
+	return fmt.Sprintf(
+		"[%s%s] [%s ⇄ %s] (ns: %d)",
+		t.Type(),
+		t.Family(),
```

Comment:
```suggestion
		t.Type().String(),
		t.Family().String(),
```

Reply:
Explicit use of `.String()` isn't necessary. See this from the `fmt` docs:
> 5. If an operand implements method String() string, that method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).

Reply:
That's neat - TIL!